azure/cpi.yml

```diff
@@ -1,3 +1,24 @@
+# $ export BOSH_LOG_LEVEL='debug'
+# $ export BOSH_LOG_PATH='~/run.log'
+# $ bosh create-env ~/workspace/bosh-deployment/bosh.yml \
```

Comment:
if you want to include example, let's just update values in test.sh to be more realistic.